### DISCUSSION TITLE
快速粘贴无法在QQ和微信上使用，大佬看看呗。

### DISCUSSION THREAD

**Discussion by Corey1984** [shuax/MouseInc.Settings#267] [Q&A]
你好，MouseInc 使用很方便，但是快速粘贴好像出现问题，按住左键点右键，无法在QQ和微信上使用快速粘贴，（快速复制可以）
![1650804326(1)](https://user-images.githubusercontent.com/104307364/164977200-4cb534bf-53b4-4bd0-9710-76676b37ec03.jpg)

**Comment by Corey1984** [comment]
补充：快速复制会偶尔失灵，均在QQ和微信对话框内。

**Comment by shuax** [comment]
部分软件不支持，我也没办法，不知道他们有什么特殊操作。

**Reply by Corey1984** [reply]
好吧谢谢。

**Reply by DeepNight-MING** [reply]
> 部分软件不支持，我也没办法，不知道他们有什么特殊操作。

大佬，好像快速粘贴功能目前就没有找到可以应用的地方，包括浏览器、聊天软件、记事本、笔记软件（Word，Obsidian，Typora）这些常用场景都用过了，都无法使用，但是Typora\Obsidian其实是支持左键点击的情况下Ctrl+V的，请问能够另外更换一个方式呢？

**Reply by DeepNight-MING** [reply]
> 部分软件不支持，我也没办法，不知道他们有什么特殊操作。

抱歉，谢谢大佬提供的软件，祝原理网络的生活愉快，万分感谢~！
